### ISSUE TITLE
Add the ability to set the hotkey for viewing the stats to a mouse button also

### ISSUE THREAD

**Issue by mrsrabs** [nxtdaydelivery/quickStats#2]
Ive changed the quickstats hotkey to a mouse button but that didnt seem to work but keyboard works as intended so it will be really helpful if you made the mod compatible with mouse buttons also, thank you for such useful mod<3

**Comment by nextdayy** [comment]
Of course! Bit busy at the moment with another project at the moment (Oneconfig), but I'll get on that as soon as I can :)
Thanks for the feedback!
